src/main.ts

```diff
@@ -24,7 +24,8 @@ const args = Args.create("chroner-collector", "A script for farming chroner", {
 export function main(command?: string) {
   Args.fill(args, command);
 
-  const completed = args.turns < 0 ? () => false : () => myAdventures() === -args.turns;
+  const turncount = myTurncount();
+  const completed = args.turns < 0 ? () => myTurncount() - turncount >= args.turns || myAdventures() === 0 : () => myAdventures() === -args.turns;
```

Comment:
Can we `prettier` this? it is hard to read as a one-liner

Reply:
Yeah I'm used to prettier on save which isn't in this repo